crates/edr_rpc_server/src/node.rs

```diff
@@ -1,28 +1,36 @@
 mod node_data;
 mod node_error;
 
-use std::mem;
+use std::{mem, sync::Arc};
 
 use edr_eth::{
+    receipt::BlockReceipt,
     remote::{
         filter::{FilteredEvents, LogOutput},
-        BlockSpec,
+        BlockSpec, BlockTag,
     },
     serde::ZeroXPrefixedBytes,
     signature::Signature,
-    Address, Bytes, U256, U64,
+    transaction::SignedTransaction,
+    Address, Bytes, B256, U256, U64,
 };
 use edr_evm::{
-    blockchain::BlockchainError,
+    blockchain::{BlockchainError, SyncBlockchain},
     state::{AccountModifierFn, StateError},
-    AccountInfo, Block, Bytecode, MineBlockResult, KECCAK_EMPTY,
+    AccountInfo, Block, Bytecode, MineBlockResult, SyncBlock, KECCAK_EMPTY,
 };
 use k256::SecretKey;
 use tokio::sync::Mutex;
 
 pub use self::node_error::NodeError;
 use crate::{filter::Filter, node::node_data::NodeData, Config};
 
+#[derive(Debug)]
```

Comment:
Should I do anything extra apart from deriving debug in a struct that's as simple as this one? It's only used to return two values from a single method.

Reply:
Just add a documentation comment above the derive `///`